lib/beaker/host/cisco.rb

```diff
@@ -108,8 +108,8 @@ def validate_setup
       msg = nil
       msg = 'Cisco hosts must be provided with a :vrf value.' unless self[:vrf]
 
-      if !msg && self[:platform] =~ /cisco-5/ && self[:user] == 'root'
-        msg = 'Cisco-5 hosts must be provided with a :user value, as they can not SSH in as root.'
+      if !msg && self[:platform] =~ /cisco-nexus/ && self[:user] == 'root'
```

Comment:
The only access to an IOS XR platform we have available to us is a VM that one of our release engineers was able to get running in our VMpooler.  That particular VM lets us use root.  I'll change this to match the Nexus.